test/parallel/taskPar/figueroa/taskParallel.chpl

```diff
@@ -225,15 +225,15 @@ proc determineWinner (player1, player2) {
   else if player1 == choices.rock then
     if player2 == choices.paper then
       declareWinner("Player 2 wins!");
-    else declareWinner("Player 1 wins!");
+  else declareWinner("Player 1 wins!");
   else if player1 == choices.paper then
     if player2 == choices.scissors then
       declareWinner("Player 2 wins!");
-    else declareWinner("Player 1 wins!");
+  else declareWinner("Player 1 wins!");
   else if player1 == choices.scissors then
     if player2 == choices.rock then
       declareWinner("Player 2 wins!");
-    else declareWinner("Player 1 wins!");
+  else declareWinner("Player 1 wins!");
```

Comment:
These three new indentations look "less correct" to me.  That said, I'm not deeply worried about this test's formatting.

Reply:
(Also: I see that the mistake was mine).

Reply:
I can fix these indents.